src/core/Akka/Actor/ActorSystem.cs

```diff
@@ -126,6 +133,8 @@ internal BootstrapSetup()
         /// </summary>
         public Option<ProviderSelection> ActorRefProvider { get; }
 
+        public ImmutableArray<IExtensionId> Extensions { get; }
```

Comment:
Need to add an XML-DOC comment here

Reply:
If the ExtensionId (the extension factory instance) should not hold any state values / fields to setup the extensions,
would it not be better, to use just the Type? 
```csharp
public ImmutableArray<Type> Extensions { get; }
```